src/main/java/de/codecrafter47/taboverlay/config/placeholder/CustomPlaceholderColorAnimation.java

```diff
@@ -55,7 +57,7 @@ void updateText() {
         if (distance.isPresent()) {
             effectiveDistance = distance.getAsInt();
         } else {
-            effectiveDistance = ChatFormat.formattedTextLength(text) / (colors.size() - 1);
+            effectiveDistance = ChatFormat.formattedTextLength(text) / ((colors.size() - 1) + (formats.size() - 1));
```

Comment:
I don't think this line makes sense.

Reply:
Wasn't sure if this mattered here, since you also used colors here...

Reply:
That line computes the distance between the colors that are part of the animation, such that it is spread out evenly. E.g. if there are two colors, then the first letter should have the first color and the last letter should have the second color. The colors of the letters in between should interpolated. Thus `effectiveDistance` should be the length of the text.